metrics/network/remote_network/README.md

```diff
@@ -34,3 +38,19 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-iperf3.sh -b -i eno1 -u tester -a 10.xxx.xxx.xxx
 
 ```
+
+## Running the `remote-networking-memory-smem.sh` test
+
+`remote-networking-memory-smem.sh` should run in host A. The script needs the following inputs to run:
+- Arguments of the test to run. In this case the argument is `-r` for RSS memory.
+- Interface name where swarm will run.
+- User of the host B.
+- IP address of the host B.
+
+The `remote-networking-memory-smem.sh` test may be run manually:
+
+```
+$ cd metrics/network/remote_network
+$ bash remote-networking-memory-smem.sh "[options]" "<interface_name>" "<user>" "<ip_address>"
```

Comment:
the actual script has `-<x>` arguments does it (like `-i <interface>`)? - in which case, can we add those `-` arguments into this example line please?